manim/communitycolors.py

```diff
@@ -0,0 +1,7 @@
+"""The colormap of manim community"""
+
+WHITE = "#ece6e2"
+GREEN = "#87c2a5"
+BLUE = "#525893"
+RED = "#e07a5f"
+BLACK = "#343434"
```

Comment:
Move it into the place where colours are defined. There can't be a file just for this. 
Move it to the Color Enum we have and name it as `LOGO_WHITE`,`LOGO_RED` etc.

Reply:
I think the community colours should be importable like
`from manim.communitycolors import *` and not in the default namespace, that's why I put it there.